client/src/PublicRequest.ts

```diff
@@ -234,6 +236,8 @@ export type KeyguardError = {
         CANCELED: 'CANCELED',
         // used to signal that the request expired
         EXPIRED: 'EXPIRED',
+        // Specifically used to trigger a redirect to a special import after returning to caller
+        RESET_PASSWORD: 'RESET_PASSWORD',
```

Comment:
```suggestion
        GOTO_RESET_PASSWORD: 'GOTO_RESET_PASSWORD',
```

Please also move this message next to the existing GOTO_CREATE message.